roles/openshift-applier/tasks/process-file.yml

```diff
@@ -13,12 +13,20 @@
       set_fact:
         file_facts: "{{ processed_file_facts|first }}"
 
+- name: Grab patch file content if action == patch
+  set_fact:
+    patch_content: "{{ lookup('file', params) }}"
```

Comment:
We'd probably want to support using a url as well as a local file. From what I know, this lookup will limit us to just a local file.

Reply:
@oybed getting back to this... do you have a clean way to do this, or should we look at adding a filter_plugin method for this?

Reply:
@oybed nvm... check out my latest commit